verilated/src/vcd.rs

```diff
@@ -24,10 +23,17 @@ mod ffi {
     }
 }
 
+#[cfg(unix)]
 fn cstr(path: &Path) -> io::Result<CString> {
+    use std::os::unix::ffi::OsStrExt;
     Ok(CString::new(path.as_os_str().as_bytes())?)
 }
 
+#[cfg(not(unix))]
+fn cstr(path: &Path) -> io::Result<CString> {
+    Ok(CString::new(path.to_str().expect("path is not valid utf-8").as_bytes())?)
```

Comment:
Why panic here if the string isn't valid utf-8, when it returns `io::Result`?

Reply:
The quick answer is that I wanted an easy fix to try and get things building, because I wasn't sure how much work it would take.  It turns out that it wasn't much!

I've updated it now to return an `io::Error::InvalidData` if you pass it a path that cannot be used.